torchtext/experimental/vectors.py

```diff
@@ -103,6 +105,7 @@ def GloVe(name="840B", unk_tensor=None, root='.data', validate_file=True):
             - 840B
             - twitter.27B
             - 6B
+        dim (int): the dimension for the GloVe dataset to load.
```

Comment:
Let's be more explicit about the dim options for each category.

Reply:
Sure. I'll list out what the possible options are.